modules/apps/forms-and-workflow/calendar/calendar-test/src/testIntegration/java/com/liferay/calendar/search/test/CalendarBookingIndexerTest.java

```diff
@@ -177,6 +179,30 @@ public void testSearch() throws Exception {
 		assertSearchHitsLength(title, 1, LocaleUtil.US);
 	}
 
+	@Test
+	public void testSearchInTrash() throws Exception {
+		setUpSearchContext(_group, _user);
+
+		String title = RandomTestUtil.randomString();
+
+		CalendarBooking calendarBooking = addCalendarBooking(
+			new LocalizedValuesMap() {
+				{
+					put(LocaleUtil.US, title);
+				}
+			});
+
+		CalendarBookingLocalServiceUtil.moveCalendarBookingToTrash(
+			TestPropsValues.getUserId(), calendarBooking);
+
+		assertSearchHitsLength(title, 0, LocaleUtil.US);
+
+		_searchContext.setAttribute(
+			Field.STATUS, new int[] {WorkflowConstants.STATUS_IN_TRASH});
+
+		assertSearchHitsLength(title, 1, LocaleUtil.US);
+	}
+
```

Comment:
Thank you, @ChrisKian!

I was quite busy today so I could not test it as extensively as I have done before. Nonetheless, nothing caught my attention so I'm confident on it. 

I forgot to ask for integration tests. My bad! To make things faster, I implemented one here. Next time, it's your turn ;)

Again, thank you for your fix! Your help is invaluable. 

PS: from January 2 onwards, I'll be working full time on the Search team, so PRs can be sent to inacionery. Any questions, just ping [the Workflow team](https://loop.liferay.com/web/guest/home/-/loop/teams/_Workflow).